cfme/control/explorer/alerts.py

```diff
@@ -31,6 +31,21 @@ class AlertFormCommon(ControlExplorerView):
     description = Input(name="description")
     active = Checkbox("enabled_cb")
     based_on = BootstrapSelect("miq_alert_db")
+
+    @View.nested
+    class evaluate(View):  # noqa
+        type = BootstrapSelect("exp_name")
+        # Real Time Performance Parameters
+        performance_field = BootstrapSelect("perf_column")
+        performance_field_operator = BootstrapSelect("select_operator")
+        performance_field_value = Input(name="value_threshold")
+        performance_trend = BootstrapSelect("trend_direction")
+        performance_time_threshold = BootstrapSelect("rt_time_threshold")
+
+        def fill(self, values):
+            new_values = dict(type=values[0], **values[1])
+            return View.fill(self, new_values)
```

Comment:
I think you should use super here, the explicit callout is used in the widgets' init to prevent inadvertent effects.